src/views/arbeidssoker/Arbeidssoker.module.css

```diff
@@ -0,0 +1,9 @@
+.arbeidssokerButtonContainer {
```

Comment:
Kan du bruke `.navigation-container` i stedet for egen css? Vi har lagt den i den globale CSSen siden den brukes mange steder: https://github.com/navikt/dp-soknadsdialog/blob/main/src/index.css#L47